### DISCUSSION TITLE
Disable title on a button?

### DISCUSSION THREAD

**Discussion by realChrisDeBon** [tui-cs/Terminal.Gui#3384] [Q&A] [answered]
I have never encountered this before, but the button Title and Text seem to be intrinsically linked? I cannot seem to get rid of the title on the button (and some other controls when they are added as subviews). I've tried so many variations:

![Untitled](https://github.com/gui-cs/Terminal.Gui/assets/97779307/ee51be7f-6615-4440-954d-88aa3f9ecb9b)

```

                viewbutton_ = new Button()
                {
                    X = Pos.Right(log) + 1,
                    Y = Pos.Bottom(mainList) + 1,
                    Text = "View",
                    ColorScheme = blackWhite,
                    AutoSize = false,
                    BorderStyle = LineStyle.Rounded,
                    NoDecorations = true,
                };
                Rectangle test = viewbutton_.Frame;
                viewbutton_.Border.Thickness.Draw(test, ""); // I've also tried null
```

```
                viewbutton_ = new Button()
                {
                    X = Pos.Right(log) + 1,
                    Y = Pos.Bottom(mainList) + 1,
                    Text = "View",
                    Title = "",
                    ColorScheme = blackWhite,
                    AutoSize = false,
                    BorderStyle = LineStyle.Rounded,
                    NoDecorations = true,
                }; // I've tried several variations where Title is set to "" or null after object is created
```
^ this is just a few, as I suspect the underlying cause is related to Border.Thickness. No matter what though, Title and Text seem intrinsically linked. I've used this library on and off for a while now but I haven't actively kept up with many of the recent changes, can someone explain what I'm missing here?

**Comment by tig** [comment]
Turn off the border.

**Comment by tig** [comment, marked as answer]
Title and Text are linked for Button and Label. 

In v2 the Title is what's used for hotkeys etc. 

I'm guessing you want a "big" button. I'll take a look at how you can have your cake and eat it too.

**Reply by tig** [reply]
See the HotKeys scenario for more.

**Reply by tig** [reply]
FWIW, once Border is refactored to use subviews instead of the current manually drawn title/frame it will be possible for you to do, essentially:

button.Border.TitleLabel.Visible = false 

Note that, unless Button is rewritten, users won't be able to click on the border to Accept the Button. 

Do you REALLY want huge buttons? In my experience screen real estate is precious in TUI apps...

**Reply by realChrisDeBon** [reply]
Gotcha. I had a feeling it was one of those things that was maybe in the process of being refactored or worked on but I just couldn't find any posts specifying so. Disabling the border worked, thank you very much.
